src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+public class Main {
+    public static void main(String[] args) {
+        Figure[] figures = new Figure[6];
+        FigureSupplier figureSupplier = new FigureSupplier();
+        for (int i = 0; i < figures.length; i++) {
+            if (i < figures.length / 2) {
+                figureSupplier.getRandomFigure().printInformation();
+                continue;
+            }
+            figureSupplier.getDefaultFigure().printInformation();
```

Comment:
Same as the previous comment, you should store the default figures in the `figures` array instead of printing them directly.